tests/Integration/Schema/Directives/CanDirectiveDBTest.php

```diff
@@ -77,14 +76,23 @@ public function testFailsToFindSpecificModel(): void
         }
         ';
 
-        $this->expectException(ModelNotFoundException::class);
         $this->graphQL(/** @lang GraphQL */ '
         {
             user(id: "not-present") {
                 name
             }
         }
-        ');
+        ')->assertJson([
+            'errors' => [
+                [
+                    'message' => 'No query results for model [Tests\Utils\Models\User] not-present',
+                ],
+            ],
+            'data' => [
+                'user' => null,
+            ],
+        ])->assertErrorCategory('graphql')
+            ->assertStatus(200);
```

Comment:
That should be the case anyhow, regardless of what error may have occurred. 

I added one more test for this [here](https://github.com/nuwave/lighthouse/pull/1228), please remove it from this test class so it stays focused.

Reply:
Do we still want to assert the `errors.message` or do you just wan't me to remove `assertErrorCategory` and `assertStatus`?